### PR TITLE
Add support for reading and printing ADC values from the cards

### PR DESCRIPTION
**Pull request by jviereck**

### CODE REVIEW

File: sdk/master_board_sdk/include/master_board_sdk/motor_driver.h [hunk 1]
```diff
@@ -18,9 +18,14 @@ class MotorDriver
 
 	void Enable();
 	void Disable();
+
+	//adc
+	float adc[2];
```
Comment (jviereck) [inline sdk/master_board_sdk/include/master_board_sdk/motor_driver.h]: Is this the proper place to put the adc measurements? They are not really part of the motor, so putting them on the `MotorDriver` class feels a bit off.

How about renaming `MotorDriver` to `MotorBoardDriver` (that widens the scope a bit)?

**Comment by thomasfla** [comment]
Yes, I think it should be part of motor drivers since it is connected to it
physically. I'm not against renaming the class, maybe MotorDriverBoard ?

Let's keep that for later, since a refactoring is planned for the python
binding next week or so..

Thanks for the contribution.

Could you confirm that the values go from 0 to about 3.3v?

Le mar. 19 nov. 2019 à 19:28, Julian Viereck <notifications@github.com> a
écrit :

**Comment by jviereck** [comment]
> Could you confirm that the values go from 0 to about 3.3v?

I am not able to confirm this.

Right now I have a slider box attached which only gives values between 0 and 1 v (that's the values we also got for the ti boards). I don't have a setup for testing the 3.3 v at hand.

Can we land this and you might look into this later?

**Comment by jviereck** [comment]
@thomasfla , Felix G told me the values returned by the adc is a percentage and therefore should always be in the range of [0, 1]. Is that true?

If so, I would adjust the PR and update the range by reverting `SPI_QN_ADC` back to `16`

**Comment by thomasfla** [comment]
Felix G. is right, the max value is 1. 
Could you change the SPI_QN_ADC back to 16 ? 
Sorry for the mislead on this, i thought it was in volts.

**Comment by paLeziart** [comment]
I switched SP_QN_ADC and UD_QN_ADC back to 16.

**Comment by paLeziart** [comment]
Sorry, I clicked on the wrong button...

**Comment by jviereck** [comment]
> Could you change the SPI_QN_ADC back to 16 ?

Yes, this PR uses the current value of `16`.

Are there any other comments or is this ready to be merged?

**Comment by paLeziart** [comment]
I will make the Python bindings to access the `adc` property and the `PrintADC` function in another pull request.

**Pull request merged**